docs/admin/hardware.rst

```diff
@@ -22,18 +22,65 @@ In order to print submissions, a supported non-networked printer is required. Su
 
 More printer options will be added in future releases.
 
+Lenovo T series Laptops
+-----------------------
+
+Lenovo ThinkPad T490 (with 8th generation Intel Core processor)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Thinkpad T490 **with an 8th generation Intel Core processor** is a recommended option for the SecureDrop Workstation. If you plan to use it, you should follow the instructions below to ensure that the BIOS is up to date and adequately configured before proceeding with the installation.
+
+
+.. caution::
+
+  There versions of the T490 with 10th generation Intel Core processors, and are at present **untested and unsupported**.
```

Comment:
Typo/sentence construction issue ("there versions"):

> There versions of the T490 with 10th generation Intel Core processors, and are at present **untested and unsupported**.

How about:

> The versions of the T490 with 10th generation Intel Core processors are at present **untested and unsupported**.

Reply:
I agree! Added this suggestion as part of review.